assets/img/sponsors/baroodycamps.svg

```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1167" height="1233" viewBox="0 0 281 297">
```

Comment:
It's way too small still.

Reply:
Preview it somewhere. The actual logo fills probably 1/30th of the image.

Reply:
@ErikBoesen I checked it and it looks fine I'll blow it up by 40 just to be sure though

Reply:
![](https://user-images.githubusercontent.com/5923662/39203941-50938cb2-47c4-11e8-9431-0a10f17a018a.png)
This is the image right now. I circled the white image so it can be easily seen. You'll notice it doesn't take too much space up, so if we tried to display it on the website it would be really hard to see...

By the way, I still have your earbuds. I will give them to you next time I see you.